Allura/setup.py

```diff
@@ -24,6 +24,8 @@
     use_setuptools()
     from setuptools import setup, find_packages
 
+from allura.version import __version__
+
 exec open('allura/version.py').read()
```

Comment:
This `exec` line gets `__version__` imported, so the regular "import" line is not needed.

This is done this way, so that it does not cause any other imports to chain in, which is nice and safe for setup.py usage.